package.json

```diff
@@ -4,13 +4,22 @@
   "description": "random-access-storage instance backed by the Chrome file system api.",
   "main": "index.js",
   "dependencies": {
+    "mutexify": "^1.3.0",
     "random-access-storage": "^1.3.0"
   },
   "devDependencies": {
-    "standard": "^11.0.1"
+    "budo": "^11.6.3",
+    "puppeteer": "^3.0.2",
+    "random-access-test": "github:random-access-storage/random-access-test",
```

Comment:
i didn't realise we had this! should we publish that to npm?

Reply:
I noticed the same. We should, I think is really helpful to test these modules.

Reply:
published